internal/core/window.rs

```diff
@@ -152,6 +154,7 @@ pub struct Window {
     scale_factor: Pin<Box<Property<f32>>>,
     active: Pin<Box<Property<bool>>>,
     active_popup: RefCell<Option<PopupWindow>>,
+    close_requested: Callback<(), CloseRequestResponse>,
```

Comment:
This works.

I would just have used a `Box<impl FnMut()>` or something like that (wrapped in RefCell Option)

But i think that's fine too. so it can stay as is.